src/ethereum/frontier/state.py

```diff
@@ -421,3 +421,23 @@ def write_code(sender: Account) -> None:
         sender.code = code
 
     modify_state(state, address, write_code)
+
+
+def create_ether(state: State, address: Address, amount: U256) -> None:
```

Comment:
Can we rename this to `add_to_account_balance` instead?

I see a use case for this method in several places.
eg. https://github.com/ethereum/execution-specs/blob/04740bec742627f4214f88723cb8a091a48d8dcc/src/ethereum/frontier/spec.py#L421-L424
The code above can be replaced by `add_to_account_balance(state, sender, gas_refund_amount)`

Reply:
That's interesting. So while the transaction is executing, the ether used to pay for gas is "destroyed" (i.e. not in the origin account, and not in the coinbase account) and then "recreated"?

Reply:
Let's just merge this and then I'll propose a change in a separate PR.